spec/spec_helper.rb

```diff
@@ -44,6 +44,15 @@ def capture(stream)
   end
   alias :silence :capture
 
+  def with_captured_stdout
```

Comment:
The name for this smells like it's duplicate functionality to the capture method above.. Anyway to make this work?

Reply:
Shit. I didn't even notice that the above function exists. This can be removed safely. Good catch.